### PR TITLE
[Docs] Tune default Platform.sh docs based on project in production

### LINKED ISSUE #8124: Docs about running Sylius in production
There are several guides online regarding how to run Symfony & Doctrine best in production with what kind of settings. I think it would be good to gather some insights from several people (I'm not running Sylius in production yet) about what are the best settings they figured out in running Sylius specific. E.g. Doctrine metadata & query cache. If it's just general Symfony / Doctrine guidelines, then maybe we should point users to these guides in order to use.

### PR DESCRIPTION
**Pull request by stefandoorn**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | fixes #8124 |
| License         | MIT |

As requested by @CoderMaggie, some additional information on the Platform.sh docs based on findings in a project running Platform.sh in production.

### CODE REVIEW

**Comment by stefandoorn** [comment]
P.S. Some parts of the example code have been removed, e.g. the cronjobs that inject the fixtures every X hours (is from the demo store).

**Comment by CoderMaggie** [comment]
To me it looks great, but I will wait for @pjedrzejewski to approve it :)

File: docs/cookbook/platform-sh.rst [hunk 1]
```diff
@@ -151,11 +153,11 @@ This file will load ``mysql`` and ``redis`` on your Platform.sh server.
 
     # .platform/services.yaml
     mysql:
-        type: mysql
-        disk: 2048
+        type: mysql:10.0
```
Comment (lchrusciel) [inline docs/cookbook/platform-sh.rst]: Doesn't this value go along with `sever_version` of doctrine dbal? And we have [5.5](https://github.com/Sylius/Sylius/blob/master/app/config/config.yml#L35) right now in a config file.

File: docs/cookbook/platform-sh.rst [hunk 2]
```diff
@@ -117,18 +122,15 @@ This is how this file should look like for Sylius:
             rm web/app_test.php
             rm web/app_test_cached.php
             rm -rf var/cache/*
-            php bin/console --env=prod --no-debug --ansi cache:clear
+            php bin/console --env=prod --no-debug --ansi cache:clear --no-warmup
+            php bin/console --env=prod --no-debug --ansi cache:warmup
             php bin/console --env=prod --no-debug --ansi assets:install
-            npm install
-            npm run gulp
+            php bin/console --env=prod --no-debug --ansi theme:assets:install
```
Comment (lchrusciel) [inline docs/cookbook/platform-sh.rst]: Shouldn't it be optional only for users of theme bundle? We can just mention about it

File: docs/cookbook/platform-sh.rst [hunk 3]
```diff
@@ -24,16 +24,16 @@ and go through checkout. Then, when you will have a project ready, give it a nam
...
 .. code-block:: yaml
 
     # .platform.app.yaml
     name: sylius
 
-    type: "php:7.0"
+    type: "php:7.1"
     build:
-        flavor: symfony
+        flavor: composer
```
Comment (lchrusciel) [inline docs/cookbook/platform-sh.rst]: Did you notice anything when made this change?

Comment (stefandoorn) [reply docs/cookbook/platform-sh.rst]: No, but Platform.sh defines 'symfony' as deprecated and advices 'composer', therefore :)

File: docs/cookbook/platform-sh.rst (hunk 2 above)
Comment (stefandoorn) [reply docs/cookbook/platform-sh.rst]: Yes, can make a comment on it indeed, it's kind of optional. Actually I don't know what happens if you don't use a theme if you run this command..

File: docs/cookbook/platform-sh.rst (hunk 1 above)
Comment (stefandoorn) [reply docs/cookbook/platform-sh.rst]: Hm, good catch. I will check that, as I have `server_version` at 5.6. I'll check that one tomorrow.

File: docs/cookbook/platform-sh.rst (hunk 3 above)
Comment (stefandoorn) [reply docs/cookbook/platform-sh.rst]: In here they use 'composer' also: https://docs.platform.sh/frameworks/symfony.html. The Symfony docs itself are behind it seems.

Comment (stefandoorn) [reply docs/cookbook/platform-sh.rst]: Ah here is actually the commit I was searching for: https://github.com/platformsh/platformsh-example-symfony/commit/3de359c88fb5316ff653f0142428e658880f1b65#diff-fe14053ff51cb358f8b709415b2a2d17. Looking at the commit message, it seems 'symfony' is an alias of 'composer'. It's probably more safe to go with 'composer'.

**Comment by stefandoorn** [comment]
Feedback processed!

**Comment by lchrusciel** [comment]
Thanks Stefan! It will help us a lot :)

**Comment by stefandoorn** [comment]
You're welcome, happy to help!

**Pull request merged**